### PR TITLE
feat(roll): roll Playwright 1.15.0-next-1631655106000

### PR DESCRIPTION
**Pull request by mxschmitt**

### CODE REVIEW

File: tests/async/test_interception.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ async def test_page_route_should_work_with_redirect_inside_sync_XHR(page, server
 
 
 async def test_page_route_should_work_with_custom_referer_headers(page, server):
-    await page.set_extra_http_headers({"referer": server.EMPTY_PAGE})
+    await page.set_extra_http_headers({"Referer": server.EMPTY_PAGE})
```
Comment (dgozman) [inline tests/async/test_interception.py:207]: Why this change? Seems suspicious. Did we have the same change upstream?

**Review by dgozman — APPROVED** [review]

File: tests/async/test_interception.py (hunk 1 above)
Comment (mxschmitt) [reply tests/async/test_interception.py:207]: Let me try to repro it upstream.

Comment (mxschmitt) [reply tests/async/test_interception.py]: fixed, was my wrong implementation!
